source/_posts/2023-10-14-home-assistant-os-release-11.markdown

```diff
@@ -0,0 +1,56 @@
+
+We’ve also worked on the landing page which is bundled with Home Assistant OS 11. The landing page is visible to the user when starting a fresh installation of Home Assistant OS the first time. It features the same new look as the Home Assistant Core onboarding flow, and tracks issues during the bootstrapping phase, automatically displaying errors if they occur during that critical setup phase.
+
+<img src='/images/blog/2023-10-14-home-assistant-os-11/home-assistant-os-11-landing-page.png' alt='Screenshot of the new Home Assistant landing page'>
+
+The new landing page shipped with Home Assistant OS 11
+
+In this month we at Nabu Casa also increased our workforce working on the Home Assistant OS team: With [Jan Čermák](https://github.com/sairon) joining, we will have more bandwidth to implement new features as well as to tackle issues reported by our community. Welcome Jan!
+
+And finally: Home Assistant OS 11 will be pre-installed in the next batch of Home Assistant Green 🎉
+
+Enjoy the latest version of Home Assistant OS!
+
+Stefan
+
+<!--more-->
+
+- [Linux' preemptible kernel configuration](#linux-preemptible-kernel-configuration)
+- [VM filesystem freeze is being relayed to Home Assistant](#vm-filesystem-freeze-is-being-relayed-to-home-assistant)
+- [Docker and containerd Upgrades](#docker-and-containerd-upgrades)
+- [More Highlights in Home Assistant OS 11](#more-highlights-in-home-assistant-os-11)
+
+
+## Linux' preemptible kernel configuration
+We've applied Linux's preemptible kernel configuration across all boards. This results in lower latencies even on busy systems (e.g. due to slow I/O operations), making your smart home even more responsive.
+
+## VM filesystem freeze is being relayed to Home Assistant
+This is a neat feature for more advanced setups based on Proxmox (or other KVM based VMs).
+Home Assistant's recorder integration uses a database underneath (by default SQLite). When  Home Assistant takes a backup, the Supervisor notifies the database engine  before copying the database files (currently this is implemented for SQLite and MariaDB). With that notification the database engine can take the necessary steps to ensure that the database files are in a consistent state before the backup is taking place.
+However, when creating a snapshot using the VM snapshot feature the database doesn’t know about this, hence the snapshot can end up with an inconsistent state of the database. On snapshot restore, the database may or may not be able to recover from that inconsistent state, which can lead to partial or even complete data loss of the recorder data.
```

Comment:
```suggestion
However, when creating a snapshot using the VM snapshot feature, the database doesn’t know about this, and the snapshot can end up with an inconsistent state of the database. On snapshot restore, the database may or may not be able to recover from that inconsistent state. This can lead to partial or even complete data loss of the recorder data.
```